<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QStringListEditWidget</class>
 <widget class="QWidget" name="QStringListEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_Title">
     <property name="text">
      <string>Title</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolButton_add">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_remove">
       <property name="text">
        <string>--</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAddItem">
   <property name="text">
    <string>addItem</string>
   </property>
   <property name="toolTip">
    <string>Add item to list</string>
   </property>
  </action>
  <action name="actionRemoveItem">
   <property name="text">
    <string>removeItem</string>
   </property>
   <property name="toolTip">
    <string>Remove Item from list</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
